### PR TITLE
feat(experimental): Add declined transactions report

### PR DESCRIPTION
**Pull request by removed-account-123**
## Summary
Adds the declined transaction report schema.

## Checklist

- [X] Writing style
- [X] API design standards

### CODE REVIEW

File: openapi/components/schemas/ReportDeclinedTransactions.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+type: object
+properties:
+  totalCount:
+    description: Total count of declined transactions.
```
Comment (noon-dawg) [inline openapi/components/schemas/ReportDeclinedTransactions.yaml:4]: nitpick: based on our style, we avoid using 'count' in the descriptions
```suggestion
    description: Total number of declined transactions.
```

File: openapi/components/schemas/ReportDeclinedTransactions.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    type: array
+    items:
+      type: object
+      properties:
+        message:
+          description: Description or message related to the declined transactions report.
+          type: string
+          example: error
+        count:
+          description: Count of declined transactions.
```
Comment (noon-dawg) [inline openapi/components/schemas/ReportDeclinedTransactions.yaml:17]:
```suggestion
          description: Number of declined transactions.
```

**Review by noon-dawg — APPROVED** [review]
Looks good on the docs side

**Comment by dariashka** [comment]
Please review @Rebilly/php @Rebilly/js

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**